subsys/bluetooth/shell/ticker.c

```diff
@@ -0,0 +1,124 @@
+/** @file
+ * @brief Bluetooth Controller Ticker functions
+ *
+ */
+
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <shell/shell.h>
+#include <misc/printk.h>
+
+#include "../controller/util/mayfly.h"
+#include "../controller/ticker/ticker.h"
+
+#define TICKER_SHELL_MODULE "ticker"
+
+#if defined(CONFIG_BLUETOOTH_MAX_CONN)
+#define TICKERS_MAX (CONFIG_BLUETOOTH_MAX_CONN + 2)
+#else
+#define TICKERS_MAX 2
+#endif
+
+static void ticker_op_done(u32_t err, void *context)
```

Comment:
ok.. something like ticker_default_op_done ? that module can reuse? I am ok with that, but in a new PR.